src/shared/components/gene-search-panel/GeneSearchPanel.module.css

```diff
@@ -94,6 +94,12 @@
   cursor: pointer;
 }
 
+.geneSymbol {
+  font-weight: var(--font-weight-bold);
+  text-transform: uppercase;
+  padding-right: 8px;
```

Comment:
Curious: do we need to customise the space to the right of the gene symbol? It is not just a regular space between words?

Reply:
XD has more gap than a regular space

Reply:
Ok. Let's do margin instead of the padding then.